src/pages/umi/web3js-differences-and-adapters.md

```diff
@@ -0,0 +1,377 @@
+
+### Umi
+```ts
+import { createUmi } from '@metaplex-foundation/umi-bundle-defaults'
+import { transfer, mplCore } from '@metaplex-foundation/mpl-core'
+
+// Generate a new Umi instance
+const umi = createUmi('https://api.devnet.solana.com').use(mplCore())
+
+// Create a new instruction (like a core nft transfer)
+// get instructions will give you an Array of instructions
+const umiInstructions = transfer(umi, {...TransferParams}).getInstructions();
+```
+
+### Web3Js
+```ts
+import { SystemProgram } from '@solana/web3.js';
+
+// Create a new instruction (like a lamport transfer)
+const web3jsInstruction = SystemProgram.transfer({...TransferParams})
+```
+
+Next, let's look into how to use the adapters.
+
+### From Web3Js to Umi
+```ts
+import { SystemProgram } from '@solana/web3.js';
+import { fromWeb3JsInstruction } from '@metaplex-foundation/umi-web3js-adapters';
+
+// Generate a new Umi instance
+const umi = createUmi('https://api.devnet.solana.com')
+
+// Create a new instruction (like a lamport transfer)
+const web3jsInstruction = SystemProgram.transfer({...TransferParams})
+
+// Convert it using the UmiWeb3jsAdapters Package
+const umiInstruction = fromWeb3JsInstruction(web3jsInstruction);
+```
+
+### From Umi to Web3Js
+```ts
+import { createUmi } from '@metaplex-foundation/umi-bundle-defaults'
+import { transfer, mplCore } from '@metaplex-foundation/mpl-core'
+import { toWeb3JsInstruction } from '@metaplex-foundation/umi-web3js-adapters';
+
+// Generate a new Umi instance
+const umi = createUmi('https://api.devnet.solana.com').use(mplCore())
+
+// Create a new instruction (like a core nft transfer)
+const umiInstruction = transfer(umi, {...TransferParams}).getInstructions();
+
+// Convert it using the UmiWeb3jsAdapters Package
+const web3jsInstruction = umiInstruction.map(toWeb3JsInstruction);
+```
+
+## Transactions
+
+The Solana runtime supports two transaction versions:
+- Legacy Transaction: older transaction format with no additional benefit
+- 0 / Versioned Transaction: added support for Address Lookup Tables
+
+**Note**: if you're not familiar with the concept of Versioned Transactions, read more about it [here](https://solana.com/it/docs/advanced/versions)
+
+For `umi` and `umi-web3js-adapters` we added support for both transaction types! 
+
+### Umi
+```ts
+import { createUmi } from '@metaplex-foundation/umi-bundle-defaults'
+import { transfer, mplCore } from '@metaplex-foundation/mpl-core'
+// Generate a new Umi instance
+const umi = createUmi('https://api.devnet.solana.com').use(mplCore())
+
+// Create a new Umi Legacy Transaction
+const umiTransaction = transfer(umi, {...TransferParams}).useLegacyVersion();
+
+// Create a new Umi Versioned Transaction
+const blockhash = await umi.rpc.getLatestBlockhash()
+
+const instructions = transfer(umi, {...TransferParams}).getInstructions()
```

Comment:
Without `.getInstructions()` it would be a transaction builder already that has what you set below automatically. No need to be that verbose.
`transfer(umi, {...TransferParams}).build(umi)` would be type `Transaction` already and by default v0 with umi.identity as payer.